internal/cel/cel.go

```diff
@@ -295,20 +306,22 @@ func (p *CELParser) CheckStepRunOutAgainstKnownV1(out *StepRunOut, knownType sql
 	return nil
 }
 
-func (p *CELParser) EvaluateBooleanExpression(expr string, inputData map[string]interface{}) (bool, error) {
-	ast, issues := p.stepRunEnv.Compile(expr)
+func (p *CELParser) EvaluateBooleanExpression(expr, eventId string, input, additionalMetadata map[string]interface{}) (bool, error) {
```

Comment:
nit: naming -- should probably be `EvaluateEventExpression` since this can't evaluate any boolean expr